themes/ropensci/layouts/partials/otherposts.html

```diff
@@ -0,0 +1,16 @@
+
+    <div class="col-offset-2 col-8 top-4 labels">
+<p>   
+{{$staff := .Site.Params.staff}}
+{{$leadership := .Site.Params.leadership}}
+{{$pages := .Site.Pages.ByPublishDate.Reverse}}
+{{ range $i, $e := .authors }}
+{{ range first 1 (where $pages ".Params.name" $e) }}
+{{$params := .Params}}
+{{ if gt (len .Pages) 1 }}
+<i> Find more content <a href="{{ .Permalink }}" target="_blank">by {{ $e }}</a>. </i>
```

Comment:
this is linking to ropensci.org right now, not to the netlify preview

Reply:
yes but I'm not sure what other variable to use to be honest https://gohugo.io/variables/page/